Lib/fontmake/font_project.py

```diff
@@ -185,16 +188,15 @@ def run_from_designspace(self, designspace_path, **kwargs):
         self.run_from_ufos(ufos, **kwargs)
 
     def run_from_ufos(
-            self, ufos, compatible=False, remove_overlaps=True, mti_source=None,
-            **kwargs):
+            self, ufos, remove_overlaps=True, mti_source=None, **kwargs):
         """Run toolchain from UFO sources to OpenType binaries."""
 
         if isinstance(ufos, str):
             ufos = glob.glob(ufos)
         if isinstance(ufos[0], str):
             ufos = [Font(ufo) for ufo in ufos]
 
-        if remove_overlaps and not compatible:
+        if remove_overlaps and not self.compatible:
```

Comment:
Can you also make sure `remove_overlaps` is turned off if compatible is turned on?  Or fail if they conflict?  Right now `remove_overlaps` is always on, right?  For GX / interpolation, we need it turned off.

Maybe instead of calling it `compatible`, we should call it something better.  But we can do that later when we figure out what to call these "GX" fonts.